### PR TITLE
Add `o-layout__overview--consistent-columns` modifier class.

### PR DESCRIPTION
**Pull request by notlee**
>Sometimes a page may have multiple overview sections. In this case
>it can appear odd to have three overview items followed by a
>seperate section with four more narrow items. To enforce that the
>items in each overview section have the same width apply the
>`o-layout__overview--max-columns` modifier class.

E.g. see https://github.com/Financial-Times/biz-ops-admin/pull/301#issuecomment-508690064

### CODE REVIEW

File: src/scss/_grid-areas.scss [hunk 1]
```diff
@@ -85,11 +85,15 @@
...
 	.o-layout__overview {
 		display: grid;
-		grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+		grid-template-columns: repeat(var(--_o-layout-overview-columns, auto-fit), minmax(250px, 1fr));
 		grid-gap: $_o-layout-gutter * 3;
 		grid-row-gap: $_o-layout-gutter;
 	}
 
+	.o-layout__overview--max-columns {
+		--_o-layout-overview-columns: auto-fill;
```
Comment (JakeChampion) [inline src/scss/_grid-areas.scss]: Can we use CSS Variables?!

**Comment by notlee** [comment]
A note on browser support of CSS Custom Properties:

The o-layout component was made for internal users with more permissive browser support requirements. Using CSS Custom Properties here doesn't change `o-layout`'s browser support, and doesn't preclude improving it.

File: src/scss/_grid-areas.scss (hunk 1 above)
Comment (notlee) [reply src/scss/_grid-areas.scss]: See https://github.com/Financial-Times/o-layout/pull/95#issuecomment-508723491 :)

**Comment by wheresrhys** [comment]
Thanks for this. My only comments is that 'max-columns' doesn't convey to me the behaviour that it implements. Maybe fix-columns would be better? Or predictable-columns. or...??

**Comment by notlee** [comment]
@wheresrhys we had thought about `--fixed` but thought it might be misleading because the column count changes according to the viewport size (drops to three, two, then one on mobile devices). `--max-columns` attempts to convey that, regardless of the number of columns, there will always be as many as there can be. I'm struggling to think of a better name for that 🤔

**Comment by notlee** [comment]
Rhys suggested `--consistent-columns` and I'm okay with that

**Pull request merged**